lib/node_modules/@stdlib/math/base/special/cceil/package.json

```
@@ -66,5 +66,138 @@
     "complex",
     "cmplx",
     "number"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "ceil",
+      "alias": "cceil",
+      "pkg_desc": "round each component of a double-precision complex floating-point number toward positive infinity",
+      "desc": "rounds each component of a double-precision complex floating-point number toward positive infinity",
+      "short_desc": "complex ceil",
```

```suggestion
      "pkg_desc": "round each component of a complex number toward positive infinity",
      "desc": "rounds each component of a complex number toward positive infinity",
      "short_desc": "",
```

This one is a bit tricky for a "short" description which is intended as a means of converting a substring within `desc` to a, e.g., a link in Markdown. Here, that isn't really possible, so we can set to an empty string.